libkbfs/config_local.go

```diff
@@ -208,6 +209,15 @@ func MakeLocalUsers(users []libkb.NormalizedUsername) []LocalUser {
 	return localUsers
 }
 
+func getDefaultCleanBlockCacheCapacity() uint64 {
+	vmstat, err := mem.VirtualMemory()
+	if err == nil {
+		// By default,  we try to cap at 1/8 of available memory
+		return vmstat.Total / 8
```

Comment:
Ah with the dirty bcache also set to the same value, I guess it makes more sense to keep `/4`. Never mind!